templates/nsis/oneClick.nsh

```diff
@@ -5,7 +5,13 @@
       !ifdef INSTALL_MODE_PER_ALL_USERS
         ${StdUtils.ExecShellAsUser} $0 "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "open" ""
       !else
-        ExecShell "" "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk"
+        ${GetParameters} $R0
+        ${GetOptions} $R0 "--update" $R1
+        ${IfNot} ${Errors}
+          ExecShell "" "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk" "--updated"
+        ${Else}
+          ExecShell "" "$SMPROGRAMS\${PRODUCT_FILENAME}.lnk"
+        ${endif}
```

Comment:
Please modify https://github.com/electron-userland/electron-builder/blob/master/templates/nsis/boringInstaller.nsh#L10 also (sorry, I didn't mentioned this place).